cognee/tests/test_neptune_analytics_graph.py

```
@@ -0,0 +1,313 @@
+
+    print("------DELETE-------")
+    # delete all nodes and edges:
+    await na_adapter.delete_graph()
+
+    # delete all nodes by node id
+    # node_ids = [str(node.id) for node in nodes]
+    # await na_adapter.delete_nodes(node_ids)
+
+    has_edges = await na_adapter.has_edges(edges)
+    if len(has_edges) == 0:
+        print(f"Delete successful")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)